WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -686,15 +682,25 @@ class MySiteViewController: UIViewController, NoResultsViewHost, UIScrollViewDel
                                     bottomAnchor: view.safeAreaLayoutGuide.bottomAnchor)
 
         if let blog = blog,
-           noResultsViewController.view.superview == nil {
+           noSitesViewController.view.superview == nil {
             createButtonCoordinator?.showCreateButton(for: blog)
         }
     }
 
 // MARK: - Add Site Alert
 
+    func didTapAddNewSiteButton() {
+        presentInterfaceForAddingNewSite()
+    }
+
+    func didTapAccountAndSettingsButton() {
+        let meViewController = MeViewController()
+        present(meViewController, animated: true)
```

Comment:
> but even with one it seems to work kind of strange

It's because I was presenting the vc - I've changed it to push the vc instead 

https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/df56555e-64a4-4f1f-b881-193f2d40ac83